packaging/ondemand-bc_classroom_mathematica

```diff
@@ -0,0 +1,46 @@
+# Disable debuginfo as it causes issues with bundled gems that build libraries
+%global debug_package %{nil}
+%global repo_name bc_classroom_stable_diffusion
+%global app_name bc_classroom_stable_diffusion
```

Comment:
@Oglopf you need to change the names of the package here otherwise you're building stable diffusion RPMs.